NBXplorer.Tests/TestConfig.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NBXplorer.Tests
+{
+	public static class AzureServiceBusTestConfig
+	{
+		public static string ConnectionString
+		{
+			get
+			{
+				//Put your service bus connection string here - requires READ / WRITE permissions
+				return "";
+			}
+		}
+
+		public static string NewBlockQueue
```

Comment:
Use public const string NewBlockQueue = "newblock";

Reply:
Actually you can remove that right? this seems to be the default

Reply:
I think these should stay in TestConfig as the Test will independently connect to the queues and does not have access to the defaults set in NBXplorer config. If  these values are hardcoded in the test then you are forced to name your queues in your test environment with the hardcoded values.